### ISSUE TITLE
hyperparameter search

### ISSUE THREAD

**Issue by dmgroppe** [dmgroppe/EPISIM#3]
learning rate, # of nodes, whitening, regularization
